PontoFacil/PontoFacil/Converters/BoolToVisibilityConverter.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Data;
+
+namespace PontoFacil.Converters
+{
+    public class BoolToVisibilityConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, string language)
+        {
+            return (((bool)value) ? Visibility.Visible : Visibility.Collapsed);
+        }
+
+        public object ConvertBack(object value, Type targetType, object parameter, string language)
+        {
+            return (((Visibility)value) == Visibility.Visible);
```

Comment:
Since value is an object, please try to check against null or use a try/catch (preferable) block and have a default value returning on the catch block